dashboard/lib/projects_list.rb

```diff
@@ -0,0 +1,41 @@
+module ProjectsList
+  class << self
+    def fetch_section_projects(section)
```

Comment:
Ah I didn't notice the privates were also class methods. In that case I don't care strongly, especially since the style guide does allow `class << self`